packages/dashboard-core-plugins/src/panels/IrisGridPanel.tsx

```diff
@@ -125,6 +126,7 @@ export interface PanelState {
 
 export interface IrisGridPanelProps {
   children?: ReactNode;
+  dh: DhType;
```

Comment:
Shouldn't this be a `makeApi` prop like you did for `ChartPanel`? Since we may need to show the panel before the API loads?